cves/CVE-2013-0870.yml

```diff
@@ -150,10 +163,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: I could not find any unit tests.
```

Comment:
They have some code coverage listed for the file the function is in, so there's probably a test somewhere. I couldn't make heads or tails of the FATE stuff, so I used this to kind of guess on mine. http://coverage.ffmpeg.org/